images/debian.yaml

```diff
@@ -831,6 +831,13 @@ files:
 
  - path: /etc/machine-id
    generator: remove
+   types:
+    - container
+
+ - path: /etc/machine-id
+   generator: dump
+   types:
+    - container
```

Comment:
You probably meant `vm` there instead of `container` and we in fact just want it to be empty in both cases, same way we've done it in ubuntu.